spec/advanced_search_spec.rb

```diff
@@ -382,27 +382,27 @@
       before(:all) do
       end
       it 'before topics selected' do
-        resp = solr_resp_doc_ids_only({ 'fq' => 'format:("Video"), language:("English"), building_facet:("Media & Microtext Center")', 'q' => 'collection:*' }.merge(solr_args))
+        resp = solr_resp_doc_ids_only({ 'fq' => 'format_main_ssim:("Video"), language:("English"), building_facet:("Media & Microtext Center")', 'q' => 'collection:*' }.merge(solr_args))
         expect(resp.size).to be >= 35_000
         expect(resp.size).to be <= 45_000
       end
       it 'add topic feature films' do
-        resp = solr_resp_doc_ids_only({ 'fq' => 'format:("Video"), language:("English"), building_facet:("Media & Microtext Center"), topic_facet:("Feature films")', 'q' => 'collection:*' }.merge(solr_args))
+        resp = solr_resp_doc_ids_only({ 'fq' => 'format_main_ssim:("Video"), language:("English"), building_facet:("Media & Microtext Center"), topic_facet:("Feature films")', 'q' => 'collection:*' }.merge(solr_args))
```

Comment:
That's weird. This change (format > format_main_ssim) was in a commit from 8 months ago.